app/views/bulk_updates/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="row row-content">
+  <section>
+    <h2>Bulk Updates for <%= @provider_id %></h2>
+
+    <table class="bulk-updates-list-table">
+      <thead>
+        <tr>
+          <th>Task ID</th>
+          <th>Status</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% if @tasks.blank? %>
+          <tr>
+            <td colspan="2">No tasks found</td>
+          </tr>
+        <% else %>
+          <% @tasks.each do |task| %>
+            <tr>
+              <td><%= link_to task.fetch('task-id', nil), bulk_update_path(task.fetch('task-id', 'not found')) %></td>
```

Comment:
For both? For the second one, in the unlikely case that there was no `task-id` I was trying to avoid the No Route error if the value passed to the route helper was `nil`